eventmesh-connector-sdks/eventmesh-connector-sdk-java/src/main/java/org/apache/eventmesh/connector/api/config/PubSubConfig.java

```diff
@@ -38,4 +38,6 @@ public class PubSubConfig {
 
     private String passWord;
 
+    private String subsystem;
```

Comment:
here don't use the subsystem, use `appId` instead which is in `PubSubConfig` class